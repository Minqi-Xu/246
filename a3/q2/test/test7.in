b 10
r
b 5
r
b 5
r
m
t
c
