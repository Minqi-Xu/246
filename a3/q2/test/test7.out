10
20
Current position: (-5,-5), Facing West
